<launch>
  <node pkg="ros_web_service" type="totwistodom_service.py" name="twist_publisher" args="" required = "true">
        <param name="robot_prefix" value="robot_"/>
        <param name="num_robots" value="4"/>
        <param name="port" value="8082"/>
        <param name="data_uri" value="/twist"/>
  </node>

  <node pkg="ros_web_service" type="totwistscan_service.py" name="twistscan_publisher" args="" required = "true">
        <param name="robot_prefix" value="robot_"/>
        <param name="num_robots" value="4"/>
        <param name="port" value="8083"/>
        <param name="data_uri" value="/twist"/>
  </node>
  
  <node pkg="ros_web_service" type="totwistodom_service.py" name="twist_publisher_global" args="" required = "true">
        <param name="robot_prefix" value="robot_"/>
        <param name="num_robots" value="4"/>
        <param name="odomname" value="/base_pose_ground_truth"/>
        <param name="port" value="8084"/>
        <param name="data_uri" value="/twist"/>
  </node>

  <node pkg="ros_web_service" type="topose_service.py" name="pose_publisher" args="" required = "true">
        <param name="robot_prefix" value="robot_"/>
        <param name="num_robots" value="4"/>
        <param name="port" value="8085"/>
        <param name="data_uri" value="/twist"/>
  </node>
  <node pkg="web_video_server" type="web_video_server" name="video_server">
        <param name="port" value="8081"/>
  </node>
  <node pkg="stage_ros" type="stageros" name="simulation" args="$(find ros_web_service)/launch/willow-erratic.world" respawn="true">
        <param name="base_watchdog_timeout" value="50"/>
  </node>
</launch>

